lib/App/index.js

```diff
@@ -277,10 +268,20 @@ class App {
         }
 
         // validate `appJson.drivers[].images`
-        if (levelPublish) {
+        if (level >= VALIDATION_LEVELS.PUBLISH) {
           await this._validateImages(driver.images, 'driver');
         }
 
+        if (level >= VALIDATION_LEVELS.VERIFIED) {
+          if (driver.platforms === undefined) {
+            throw new Error(`drivers.${driver.id}: property \`platforms\` is required in order to publish a verified app.`);
+          }
+
+          if (driver.connectivity === undefined) {
+            throw new Error(`drivers.${driver.id}: property \`connectivity\` is required in order to publish a verified app.`);
```

Comment:
This doesn't verify the value of these properties

Reply:
The schema takes care of that, the schema just doesn't make sure the property actually exists.